src/System.IO.Ports/tests/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netfx;
+      uap-Windows_NT;
```

Comment:
I don't think you need the new configuration, looks like you can keep this as netstandard and that should work since you are using platform detection to check if its uap or not.